Add Root URL info under License file management

adeverteuil opened the pull request:
Edited through GitHub, not tested on development web server.

**What this PR does / why we need it**:

This is a common confusion for Enterprise customers.

**Special notes for your reviewer**:

You can refer to Alexandre on #customer-support on Slack for more information.

adeverteuil committed:
Add Root URL info under License file management

Edited through GitHub, not tested on development web server.

oddlittlebird commented on docs/sources/enterprise/_index.md:
```diff
@@ -86,3 +86,20 @@ license_path = /company/secrets/license.jwt
+
+### Root URL
+
+As part of the validation checks at startup, Grafana will compare the license URL to the [`root_url`](../installation/configuration/#root-url) in your configuration. Make sure to update the [`root_url`](../installation/configuration/#root-url) in your configuration. It should be the URL that users type in their browsers to access the frontend, not the node hostname(s).
```
```suggestion
Update the [`root_url`](../installation/configuration/#root-url) in your configuration. It should be the URL that users type in their browsers to access the frontend, not the node hostname(s).

This is important, because as part of the validation checks at startup, Grafana compares the license URL to the [`root_url`](../installation/configuration/#root-url) in your configuration. 
```

oddlittlebird replied:
Try to write in present tense. "Compares" rather than "will compare."

oddlittlebird commented on docs/sources/enterprise/_index.md:
```diff
@@ -86,3 +86,20 @@ license_path = /company/secrets/license.jwt
+
+As part of the validation checks at startup, Grafana will compare the license URL to the [`root_url`](../installation/configuration/#root-url) in your configuration. Make sure to update the [`root_url`](../installation/configuration/#root-url) in your configuration. It should be the URL that users type in their browsers to access the frontend, not the node hostname(s).
+
+In `grafana.ini`:
```
```suggestion
In your configuration file:
```

oddlittlebird replied:
Depending on the system, it might be custom.ini rather than grafana.ini. I recommend adding a link to #https://grafana.com/docs/grafana/latest/installation/configuration/#config-file-locations, which outlines where users should edit their config files.

oddlittlebird commented on docs/sources/enterprise/_index.md:
````diff
@@ -86,3 +86,20 @@ license_path = /company/secrets/license.jwt
+root_url = https://grafana.blah.com/
+```
+
+Or with the environment variable:
````
```suggestion
Or with an environment variable:
```

oddlittlebird reviewed (approved):
Some minor editing suggestions.

aknuds1 commented:
@adeverteuil You want to review @oddlittlebird's suggestions?

adeverteuil committed:
Update docs/sources/enterprise/_index.md

Make updating root_url instructions more straightforward, with action first and explanation second.

Co-authored-by: Diana Payton <52059945+oddlittlebird@users.noreply.github.com>

adeverteuil committed:
Update docs/sources/enterprise/_index.md

Co-authored-by: Diana Payton <52059945+oddlittlebird@users.noreply.github.com>

adeverteuil committed:
Update docs/sources/enterprise/_index.md

Co-authored-by: Diana Payton <52059945+oddlittlebird@users.noreply.github.com>

adeverteuil commented:
Looks good, thanks for the suggestions!

aknuds1 committed:
Merge remote-tracking branch 'origin/master' into adeverteuil-patch-1
